src/problem-analysis/state-of-the-art.tex

```diff
@@ -0,0 +1,85 @@
+% textidote: ignore begin
+\section{State of the Art}\label{sec:state-of-the-art}
+% textidote: ignore end
+
+The following section will look at and analyze existing solutions and their approach to visualizing statistics and
+analytics, while looking for inspirations or areas to improve upon.
+The chosen solutions are \acrlong{ga4} and Microsoft BI, as they are both widely used for commercial purposes and
+provide a wide range of features.
+% TODO change microsoft bi to a different analytics tool that could fit better for the solution
+
+\subsection{Google Analytics}\label{subsec:google-analytics}
+
+% textidote: ignore begin % textidote doesn't like it when sentences start with acronyms
```

Comment:
@Danar435 This is because you are supposed to spell it out unless you are using it as an acronym... Imagine someone started a sentence like "GCD is a mathematical term for..." instead of "The greatest common divisor can be described as..."